src/euphorie/client/browser/templates/password_recovery_email.pt

```diff
@@ -14,9 +14,8 @@
     <tal:i18n i18n:name="reset_url">${here/absolute_url}/passwordreset/${randomstring}</tal:i18n></tal:i18n>
 
   <tal:i18n i18n:translate="mailtemplate_text_expirationdate_linkreset">(This link is valid for
-    <span tal:replace="plone_mail_password_view/expiration_timeout"
```

Comment:
Given you removed the use of this view, please check if the definition of `plone_mail_password_view` should be removed.